From 47d5121ae6ec33de1d2fe48d5700999ec0b685da Mon Sep 17 00:00:00 2001
From: "build@apk-groulx" <build@apk-groulx.praxis>
Date: Sat, 31 Jul 2021 21:44:44 +0000
Subject: [PATCH 1/1] Remove runtime install

---
 build.sh | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/build.sh b/build.sh
index b2fe4ad..99f8923 100755
--- a/build.sh
+++ b/build.sh
@@ -125,9 +125,6 @@ fi
 set -x
 scriptroot="$( cd -P "$( dirname "$source" )" && pwd )"
 
-# runtime 3.1.1 required for darc
-"$scriptroot/eng/common/dotnet-install.sh"  -runtime dotnet -version 3.1.1
-
 if [ "$alternateTarget" == "true" ]; then
   "$CLIPATH/dotnet" $SDKPATH/MSBuild.dll "$scriptroot/build.proj" /bl:source-build-test.binlog /flp:v=diag /clp:v=m "$@"
 else
-- 
2.32.0

