From c82d6ad7fa2dfbfa4168adb6b6adf1c5c1b1214d Mon Sep 17 00:00:00 2001
From: "build@apk-groulx" <build@apk-groulx.praxis>
Date: Sat, 17 Jul 2021 14:28:50 +0000
Subject: [PATCH 1/1] hardcode illink target

---
 eng/illink.targets | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/eng/illink.targets b/eng/illink.targets
index 928e9934858..3aff0f38dbd 100644
--- a/eng/illink.targets
+++ b/eng/illink.targets
@@ -289,8 +289,8 @@
             RootAssemblyNames=""
             OutputDirectory="$(ILLinkTrimOutputPath)"
             ExtraArgs="$(ILLinkArgs)"
-            ToolExe="$(_DotNetHostFileName)"
-            ToolPath="$(_DotNetHostDirectory)" />
+            ToolExe="dotnet"
+            ToolPath="$(DotNetRoot)" />
 
   </Target>
 
-- 
2.30.2

